RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -257,7 +262,9 @@ void PixelThresholdClusterizer::copy_to_buffer(DigiIterator begin, DigiIterator
   for (DigiIterator di = begin; di != end; ++di) {
     int row = di->row();
     int col = di->column();
-    int adc = electron[i++];  // this is in electrons
+    // VV: do not calibrate a fake pixel, it already has a unit of 10e-:
+    int adc = (di->flag() != 0) ? di->adc() * 10 : electron[i];  // this is in electrons
```

Comment:
this line is triggering an issue in the static analyzer
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-546425/17312/llvm-analysis/report-7cad71.html#EndPath

somehow the old variant was not triggering an issue. So, it's apparently a false-positive. Perhaps compute smth like `int adcElectron = electron[i++];` and then use it here in the ternary.

Reply:
probably because the default initialization of electron[] was commented out. For me, it is only formally incorrect, because it does not make a difference if we use 0 or a random value for pixel charge if otherwise electron[] is not correctly set before using it. I will just uncomment it as in the original code to initialized it to 0.